### ISSUE TITLE
Could be possible to have a pre-compiled version for windows so it is easy to install in a server?

### ISSUE THREAD

**Issue by eblumenfeld** [tojocky/node-printer#70]
This is just a suggestion:
Some Windows packages come with the pre-compiled version, due to the hassle of the compile cycle in the os (need to install python an a working sdk or visual studio, which is not a small task, specially in a windows server).

Some packages, like:
https://github.com/tjanczuk/edge
include the precompiled .node in the package (even 32 and 64 bits version).

could be possible to do that here?

Thank you,

**Comment by tojocky** [comment]
yes, I started to do it in https://github.com/tojocky/node-printer-bin but need to maintain.

**Comment by eblumenfeld** [comment]
are you publishing that version to npm or has to be installed from github
directly?


Eduardo Blumenfeld
ed@gotoitsolutions.com
o: 954-892-5878

**Comment by tojocky** [comment]
Not yet.

**Comment by masterofdaemon** [comment]
you can make docker package of your module using node and cups docker containers (for windows or custom linux)

**Comment by sultan99** [comment]
When will it be ready?
